apps/pm-user/src/components/organisms/quickLinkCard/QuickLinkCard.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import styles from './quickLinkCard.module.scss';
+import { MyCard } from '../../molecules';
+
+function QuickLinkCard() {
```

Comment:
QuickLinksCard would be a better name for this component because we will show multiple items inside it.
So, please consider renaming this to QuickLinksCard

Reply:
this issue is resolved